ingestion-beam/pom.xml

```diff
@@ -17,9 +17,9 @@
 
         <!-- Keep these dependency versions in sync with those pulled in by beam;
              check https://mvnrepository.com/artifact/org.apache.beam -->
-        <beam.version>2.10.0</beam.version>
+        <beam.version>2.11.0</beam.version>
         <google-cloud.version>1.49.0</google-cloud.version>
```

Comment:
@jklukas this version is used by `google-cloud-biqquery`, `google-cloud-pubsub`, and `google-cloud-storage`, but none of those are used by apache beam, afaict.

Reply:
Yes, that is correct. Beam's google lib uses only the api client libraries, not the higher-level `*-cloud` libraries.